src/app/(Authentication)/Login/Password/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function LoginScreen() {
 
   async function signIn() {
     const isPassword = await sessionHandler.signInWithEmail(email, password);
-    if (!isPassword) {
+    if (isPassword.error != null) {
```

Comment:
Let's try and trigger different types of errors from supabase and print them to the console. I imagine for a function like `signInWithEmail`, it'd error if the password is incorrect but also if there was a server/connection error. We don't want users convinced that there password is incorrect when in reality supabase servers are down.

Reply:
ok, for something like Login/Email, I left the error message hardcoded since the underlying supabase call is a SELECT, which won't properly display the error we want. just a side note lol